### PR TITLE
BAEL-95490 Review warnings in kotlin-build job

### PR DESCRIPTION
**Pull request by timis1**

### CODE REVIEW

**Review by alvin577austria — CHANGES_REQUESTED** [review]
Some comments.

File: core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/underscoreoperator/UnderScoreOperator.kt [hunk 1]
```diff
@@ -2,21 +2,24 @@ package com.baeldung.underscoreoperator
 // Before Kotlin 1.7
 class Box<T>(value: T) {
     // Some class implementation
+    var a = value
```
[the remark this answers is not in the document]
Comment (timis1) [reply core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/underscoreoperator/UnderScoreOperator.kt:5]: The variable a is an attribute to the class. So we cannot print a variable directly where we declare attributes.

File: core-kotlin-modules/core-kotlin-advanced-2/src/test/kotlin/com/baeldung/serialization/defaultValueParameters/DefaultValueSerializationUnitTest.kt [hunk 2]
```diff
@@ -1,10 +1,7 @@
 package com.baeldung.serialization.defaultValueParameters
 
-import kotlinx.serialization.EncodeDefault
-import kotlinx.serialization.Serializable
+import kotlinx.serialization.*
```
[the remark this answers is not in the document]
Comment (timis1) [reply core-kotlin-modules/core-kotlin-advanced-2/src/test/kotlin/com/baeldung/serialization/defaultValueParameters/DefaultValueSerializationUnitTest.kt:3]: I added as an individual import.

File: core-kotlin-modules/core-kotlin-lang-2/src/test/kotlin/com/baeldung/late/LateInitUnitTest.kt [hunk 3]
```diff
@@ -1,9 +1,7 @@
 package com.baeldung.late
 
 import org.junit.jupiter.api.Test
-import kotlin.test.assertFalse
-import kotlin.test.assertTrue
-import kotlin.test.assertFailsWith
+import kotlin.test.*
```
[the remark this answers is not in the document]
Comment (timis1) [reply core-kotlin-modules/core-kotlin-lang-2/src/test/kotlin/com/baeldung/late/LateInitUnitTest.kt:4]: I added as an individual import.

File: core-kotlin-modules/core-kotlin-lang/src/main/kotlin/com/baeldung/operators/Money.kt [hunk 4]
```diff
@@ -1,5 +1,6 @@
 package com.baeldung.operators
 
+import kotlinx.coroutines.newFixedThreadPoolContext
```
[the remark this answers is not in the document]
Comment (timis1) [reply core-kotlin-modules/core-kotlin-lang/src/main/kotlin/com/baeldung/operators/Money.kt:3]: I removed that useless import

File: core-kotlin-modules/core-kotlin-lang/src/test/kotlin/com/baeldung/equalityoperators/EqualityUnitTest.kt [hunk 5]
```diff
@@ -1,35 +1,34 @@
 package com.baeldung.equalityoperators
 
 import org.junit.jupiter.api.Test
-import kotlin.test.assertFalse
-import kotlin.test.assertTrue
+import kotlin.test.*
```
[the remark this answers is not in the document]
Comment (timis1) [reply core-kotlin-modules/core-kotlin-lang/src/test/kotlin/com/baeldung/equalityoperators/EqualityUnitTest.kt:4]: I added individual imports

File: pom.xml [hunk 6]
```diff
@@ -185,6 +185,7 @@
                 <executions>
                     <execution>
                         <id>compile</id>
+						<phase>compile</phase>
```
[the remark this answers is not in the document]
Comment (timis1) [reply pom.xml:188]: Fixed the indentation

**Review by alvin577austria — APPROVED** [review]

**Pull request merged**